Topics/Software_Engineering/Retro.md

```diff
@@ -0,0 +1,88 @@
+
+Once tasks and project goals were discussed, open up the meeting for discussions. This should take up the bulk of your meeting. Ask the team what went wrong during sprint? What went right? If you have comments and concerns that were brought up before the meeting, introduce them to the team and ask them if there is a current solution to said issue. Always look to get context about any issues, however, as making assumptions can easily lead to miscommunication and misunderstandings.
+
+If you have any issues or concerns brought up from previous retros, now is the time to ask the team on whether any potential solutions were thought up of that could help alleviate those issues.
+
+Don't forget to celebrate any successes that occur, no matter how small. It's important for the team to understand what was done that was successful, so that those actions can become reproduceable!
+
+#### Issue Example
+For example, say that PRs tend to take a long while to be reviewed and completed, ask the team if there were any concrete examples of PRs that were up for a long time. Ask them the context behind that PR; Why was is up for so long? Was there someone responsible for reviewing it? Were there issues with that PR? Did team members know that there was a PR?
+
+Have the team come up with quick ideas to solve this problem? Is there a communication problem that needs to be addressed? Is there a problem with the way PRs are being reviewed? Are there not enough people taking time to review the PRs?
+
+The important thing is to look for solutions that work best for your team. There is never a solution that works for every team. For this example, sometimes people need to be randomly assigned to a PR, other times, people need to promise to carve out time to address a PR. Sometimes, it's just an etiquette review to help people figure out the right and wrong way to go about reviewing a PR. Have your team discuss what might be causing an issue.
+
+Make sure that you allow time for problems to be addressed, not a single problem. It is the person leading the retro's responsibility to table any discussions that should be addressed in a separate meeting. Though retros are meant to do so, sometimes problems are large enough to warrant their own separate meeting.
+
+### Wrapping up the meeting
+
+Make sure that you give the team a chance to discuss any problems and to have them aware of them. Record any problems or issues, and their potential solutions. They should be brought up again during the next retro to see if any effort was made to address them. It is a good idea to keep this data in a living document, so that team has access to this information.
+
```

Comment:
You could further elaborate on setting actionable follow-up steps and accountability measures post-retro. This would ensure that the discussions lead to tangible improvements in subsequent sprints.

Reply:
Retros are not really meant for accountability, but rather an examination of a sprint's overall objectives. Accountability would more be on the lines of a Scrum standup, whereas a Retro is more seeing if the workload and the objectives of the sprint were successful or not.

I don't think I should add those ideas onto the section, as I feel we should focus on examining what is a retro, instead of what is not a retro.